src/plugins/remote-loader/__tests__/index.test.ts

```diff
@@ -339,11 +351,105 @@ describe('Remote Loader', () => {
           },
         ],
       },
+      {},
       {}
     )
 
     expect(plugins).toHaveLength(1)
     expect(plugins).toEqual(expect.arrayContaining([validPlugin]))
     expect(console.warn).toHaveBeenCalledTimes(1)
   })
+
+  it('accepts settings overrides from merged integrations', async () => {
+    const cdnSettings = {
+      integrations: {
+        remotePlugin: {
+          name: 'Charlie Brown',
+          version: '1.0',
+        },
+      },
+      remotePlugins: [
+        {
+          name: 'remotePlugin',
+          libraryName: 'testPlugin',
+          url: 'cdn/path/to/file.js',
+          settings: {
+            name: 'Charlie Brown',
+            version: '1.0',
+            subscriptions: [],
+          },
+        },
+      ],
+    }
+
+    const userOverrides = {
+      remotePlugin: {
+        name: 'Chris Radek',
+      },
+    }
+
+    await remoteLoader(cdnSettings, userOverrides, {
+      remotePlugin: {
+        ...cdnSettings.integrations.remotePlugin,
+        ...userOverrides.remotePlugin,
+      },
+    })
+
+    expect(pluginFactory).toHaveBeenCalledTimes(1)
+    expect(pluginFactory).toHaveBeenCalledWith(
+      expect.objectContaining({
+        name: 'Chris Radek',
+        version: '1.0',
+        subscriptions: [],
+      })
+    )
+  })
+
+  it('accepts settings overrides from options (AnalyticsBrowser)', async () => {
+    const cdnSettings = {
+      integrations: {
+        remotePlugin: {
+          name: 'Charlie Brown',
+          version: '1.0',
+        },
+      },
+      remotePlugins: [
+        {
+          name: 'remotePlugin',
+          libraryName: 'testPlugin',
+          url: 'cdn/path/to/file.js',
+          settings: {
+            name: 'Charlie Brown',
+            version: '1.0',
+            subscriptions: [],
+          },
+        },
+      ],
+    }
+
+    const initOptions: InitOptions = {
+      integrations: {
+        remotePlugin: {
+          name: 'Chris Radek',
+        },
+      },
+    }
+
+    await AnalyticsBrowser.load(
```

Comment:
I wanted to test this at the AnalyticsBrowser.load() level as well to make sure the integration options merging is working correctly.